Formula/cpputest.rb

```diff
@@ -12,16 +13,24 @@ class Cpputest < Formula
     sha256 "dbde383c51725ef3fb71480f59627e64158c5639cb827e76d7c944b0d354fa9b" => :mountain_lion
   end
 
-  head do
-    url "https://github.com/cpputest/cpputest.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
+  depends_on "cmake" => :build
 
   def install
-    system "./autogen.sh" if build.head?
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    cd "build" do
```

Comment:
Yes, which is probably the root of the reason it otherwise fails on a case-insensitive volume, though I've not proven that yet.